packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -420,7 +425,27 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
   }
 
   @autobind
-  private _onDropdownKeyDown(ev: React.KeyboardEvent<HTMLElement>) {
+  private _onDropdownBlur(ev: React.FocusEvent<HTMLDivElement>) {
+    if (this.state.isOpen) {
+      // Do not onBlur when the callout is opened
+      return;
+    }
+    if (this.props.onBlur) {
+      this.props.onBlur(ev);
+      if (ev.preventDefault) {
+        return;
+      }
```

Comment:
Because we can't do onBlur on dropdown right now.  In client form, we have display and edit modes.  If you click the field it will change to edit mode, and then click another place that is not in drop down (even if no selection has been made), it should go back to display mode.  However, there is no way to go back to display mode without making a selection right now.